src/core/algorithms/ucc/hyucc/hyucc.h

```diff
@@ -13,7 +13,7 @@ namespace algos {
 
 class HyUCC : public UCCAlgorithm {
 private:
-    std::unique_ptr<ColumnLayoutRelationData> relation_;
+    std::shared_ptr<ColumnLayoutRelationData> relation_;
```

Comment:
No need to change this
```suggestion
    std::unique_ptr<ColumnLayoutRelationData> relation_;
```